mkosi/config.py

```diff
@@ -651,7 +651,7 @@ class MkosiArgs:
     verb: Verb
     cmdline: list[str]
     force: int
-    directory: Optional[Path]
+    directory: Path
```

Comment:
This is actually a bug, it's very much intentional that the directory can be set to something that makes sure we don't parse any configuration files. So we should fix this so that the directory can be set to the empty string again

Reply:
So I checked and if I use `--directory ""` then `args.directory` will be `None` so this is working as expected. I would just drop this commit

Reply:
Dropped.

Reply:
> So I checked and if I use --directory "" then args.directory will be None so this is working as expected. I would just drop this commit

That's what I didn't expect, because `Path("")` will be the current working directory.